### PR TITLE
CEDS-2066 Remove Arrival reference screen from the Arrival journey

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

**Comment by PatrykRudnicki** [comment]
test this please

File: app/models/requests/MovementRequest.scala [hunk 1]
```diff
@@ -25,7 +25,6 @@ case class MovementRequest(
   consignmentReference: ConsignmentReferences,
   movementDetails: MovementDetailsRequest,
   location: Option[Location] = None,
-  arrivalReference: Option[ArrivalReference] = None,
```
Comment (MaciejRewera) [inline app/models/requests/MovementRequest.scala]: Should this be also propagated to the BE ?

Comment (PatrykRudnicki) [reply app/models/requests/MovementRequest.scala]: Backend is ready and merged

**Comment by daveygit2050** [comment]
test this please

**Comment by PatrykRudnicki** [comment]
@MaciejRewera This field has been removed on backend

**Pull request closed without merging**